### DISCUSSION TITLE
Community Standards

### DISCUSSION THREAD

**Discussion by elroykanye** [osscameroon/tg-gamebot#32] [Ideas]
@Mankavelda @andrew21-mch  , we will use to discuss the files which we will add according to Github community standards. The organisation has some cool templates, as well as auth0/open-source-template

**Comment by Mankavelda** [comment]
Alright, I get

**Reply by elroykanye** [reply]
Okay, so what will you do?

**Reply by Mankavelda** [reply]
as in?

**Comment by Mankavelda** [comment]
hey Vince!! please I'm waiting on you

**Reply by andrew21-mch** [reply]
Hi @Mankavelda you could so some research to see what community standards are on github,
remember google is your friend

**Reply by andrew21-mch** [reply]
http://sigdelsanjog.com.np/github-introduces-community-standards/
i find this helpful

**Reply by Mankavelda** [reply]
Thank you
